src/org/labkey/test/tests/SampleTypeRenameTest.java

```diff
@@ -173,6 +179,9 @@ public void testCustomViewWithSampleTypeRename()
                 .verifyTrue(String.format("Doesn't look like custom view '%s' was saved. Fatal error.", customViewName),
                 menuItems.contains(customViewName));
 
+        // Dismiss the menu so it doesn't get in the way.
+        menu.findVisibleMenuItems().get(0).sendKeys(Keys.ESCAPE);
```

Comment:
Would the menu's collapse method work?
```suggestion
        // Dismiss the menu so it doesn't get in the way.
        menu.collapse();
```

Reply:
Changed.